LEARNER-431, UI issues on Register and Sign In screens.

salman2013 opened the pull request:
### Description

[LEARNER-431](https://openedx.atlassian.net/browse/LEARNER-431)

Update label text in register screen and adjust the spaces between button and label text in sign in screen.

### How to test this PR
- Go on register screen and verify the labels text
- Go on sign in screen and verify the space between facebook register button and text above and below it.

salman2013 commented:
@saeedbashir Please review this code. Thanks

saeedbashir commented:
> 5b) that the space above the two buttons seems non-existent as well, and could use just a bit more spacing to emulate the register iOS / register Android screen. Perhaps it just looks closer because the text is centered aligned, but still.

You need to address Marco's comment.

salman2013 commented:
@saeedbashir I have fixed the above mention point on register screen. Please review this. Thanks

salman2013 commented:
![simulator screen shot 24-apr-2017 1 21 02 pm](https://cloud.githubusercontent.com/assets/4454296/25338795/ecb1252a-2919-11e7-9b41-d199cc252132.png)
![simulator screen shot 24-apr-2017 1 21 25 pm](https://cloud.githubusercontent.com/assets/4454296/25338796/ecb8d7d4-2919-11e7-9caf-28c50657b4a6.png)

@lizcohen
I added space under social button by 12px 
I added the space between "Forgot your password?" and the Sign in button by 20px.
Please take a look at the images and share feedback if these seems good to you.

lizcohen commented:
@salman2013 looks great, thanks! 👍

saeedbashir commented on Source/en.lproj/Localizable.strings:
```diff
@@ -553,9 +553,9 @@
+"REGISTRATION_SIGN_UP_PROMPT"="Register with";
 /* Heading on registration screen allowing users to sign up using external services like Facebook and Google. Will be preceded by text "Sign up with" */
-"REGISTRATION_SIGN_UP_ALTERNATE_PROMPT"="or sign up with email";
+"REGISTRATION_SIGN_UP_ALTERNATE_PROMPT"="or register with email";
```
@catong iOS has changed `sign up` text to `register`.

saeedbashir commented on Source/en.lproj/Localizable.strings:
```diff
@@ -553,9 +553,9 @@
 "REGISTRATION_SHOW_OPTIONAL_FIELDS"="Show optional fields";
 /* Heading on registration screen allowing users to sign up using external services like Facebook and Google. Will be followed by text "or sign up with an email". */
-"REGISTRATION_SIGN_UP_PROMPT"="Sign up with";
+"REGISTRATION_SIGN_UP_PROMPT"="Register with";
```
@catong iOS has changed `sign up` text to `Register`.
